client/src/App.js

```diff
@@ -1,32 +1,36 @@
-import { Switch, Route, useHistory } from 'react-router-dom'
+import { Switch, Route } from 'react-router-dom'
+import { useEffect, useState } from 'react'
 
 import styled from 'styled-components/macro'
-import useLocalStorage from './hooks/useLocalStorage'
 
 import CreatePage from './pages/CreatePage'
 import UsersPage from './pages/UsersPage'
 import HomePage from './pages/HomePage'
 import NavBar from './components/NavBar'
 
+const axios = require('axios')
+
 function App() {
-  const [users, setUsers] = useLocalStorage('users', [])
-  const history = useHistory()
+  const [users, setUsers] = useState([])
+
+  useEffect(() => {
+    fetch('/api/users')
+      .then(res => res.json())
+      .then(users => setUsers(users))
+      .catch(error => console.error(error))
```

Comment:
At some point you could add error handling so that the error won't be logged out but displayed for the user in the frontend.